javaparser-symbol-solver-testing/src/test/java/com/github/javaparser/symbolsolver/Issue1485.java

```diff
@@ -26,7 +27,7 @@ public void issue1485withoutSpecifyingJARs() throws IOException {
 
         CombinedTypeSolver typeSolver = new CombinedTypeSolver();
```

Comment:
Perhaps we could have an helper method in an ancestor test class that just create this combined type solver with reflection + the Jptypesolver

Reply:
I'm wary of abstraction in test code, so I'll leave that to you.